rfcs/2021-08-13-8025-internal-tracing.md

```diff
@@ -4,6 +4,19 @@ This RFC discusses an integration with the Datadog APM product for Vector's
 internal tracing data, with the goal of enabling faster and more accurate
 diagnosis of performance issues in deployed environments.
 
+## Status
+
+The team decided not to move forward with this change. Implementing tracing
+within Vector is not a worthy goal in itself, and there is a clearer, cheaper
+path to addressing the pain points laid out in this RFC via more granular timing
+metrics.
```

Comment:
Is it worth noting the eBPF discussion as well?

Reply:
I've opened a confluence page where some more detailed discussion can live. Not linked here because it's internal.